src/BlazorUI/Bit.BlazorUI/Components/Buttons/MenuButton/BitMenuButton.scss

```
@@ -21,6 +31,23 @@
 
+.bit-mnb-flw {
+    width: 100%;
+}
+
+.bit-mnb-spn {
+    width: 1em;
+    height: 1em;
+    border-radius: 50%;
+    box-sizing: border-box;
+    border-color: $clr-brd-sec;
+    border-top-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)